test/unit/lib/whitehall/asset_manager_storage_test.rb

```diff
@@ -53,22 +53,6 @@ class Whitehall::AssetManagerStorage::FileTest < ActiveSupport::TestCase
     assert_equal "http://assets-host/media/asset_manager_id/sample.docx", model.file.url
   end
 
-  test "returns file url using asset_manager_id when the model has an asset variant" do
```

Comment:
Should work: 
```
  test "returns file url using asset_manager_id when the model has an asset variant" do
    model = build(:image_data)
    model.save!
    model.reload

    assert_equal "http://assets-host/media/asset_manager_id_s216/s216_#{model.filename}", model.file.url(:s216)
  end
```

The original itself is technically a variant too, but i'd keep both tests. 

Info dump: I think at some point we had two big test `context`s going, for `AttachmentData` and `ImageData` respectively, but it was a bit repetitive, albeit thorough. There isn't much difference in how the models behave in the context of this class, so happy to leave this as is without adding in more `ImageData` or other `_Data` tests.

Reply:
Thank you - that's the winning combination of code I couldn't quite get to earlier! ⭐ Have replaced my commit.